src/main/java/com/teragrep/pth_06/planner/walker/conditions/ValidElement.java

```diff
@@ -100,4 +102,9 @@ public boolean equals(final Object object) {
         boolean equalValue = this.element.getAttribute("value").equals(cast.element.getAttribute("value"));
         return equalName && equalOperation && equalValue;
```

Comment:
equal() and hashCode calculation methods differ

Reply:
changed methods to use only element field